jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt

```diff
@@ -535,6 +532,37 @@ class CodeWhispererService {
                     null
                 }
             }
+        }?.also {
+            if (it.isProcessTimeout) {
+                LOG.debug { "Supplemental context fetch timed out in ${it.latency}ms" }
+            } else {
+                LOG.debug { "Supplemental context fetch succeeded in ${it.latency}ms" }
+                LOG.debug {
+                    """IsUtg = ${it.isUtg},
+                        Latency = ${it.latency},
+                        ContentLength = ${it.contentLength},
+                        numberOfChunk = ${it.contents.size}
+                    """.trimIndent()
+                }
+
+                if (it.contents.isNotEmpty()) {
+                    it.contents.forEachIndexed { index, chunk ->
+                        LOG.debug {
+                            """
+                            |---------------------------------------------------------------
+                            | Chunk $index:
+                            |    isUtg = ${it.isUtg},
+                            |    path = ${chunk.path},
+                            |    score = ${chunk.score},
+                            |    content = ${chunk.content}
+                            |----------------------------------------------------------------
+                            """.trimMargin()
```

Comment:
is ok but I don't usually recommend custom styles for the logs, just log them without styles is fine.